### DISCUSSION TITLE
Interface

### DISCUSSION THREAD

**Discussion by neeoder** [svsdval/video2midi#41] [Q&A]
I don't quite understand how to use the interface despite the guide, can anyone help?

**Comment by svsdval** [comment]
First you need to understand that there are fields for reading notes (they are drawn as squares), they need to be arranged so that the positions of the keys coincide with these fields, for this you need to press the right mouse button and set their general position, after using the mouse wheel and arrows on the keyboard you need to set the desired spacing between keys.


Next, in the color selection window, you need to select any channel (by default, 1 channel is associated with two colors). After that, you will need to press the control key on the keyboard and select the desired color for the association between the active (pressed piano key) color of the key and the channel with the left mouse button.

After that, you can set the tempo and press F2 / save settings and start recreating (after which the program will be closed if auto-closing of the program is selected) and a midi file will appear in your video directory
